### ISSUE TITLE
Some ui menu dropdown icon item breaks

### ISSUE THREAD

**Issue by mlopezgva** [fomantic/Fomantic-UI#742]
# Bug Report

## Steps to reproduce
1. Fomantic 2.7.2 from CDN
2. Webkit broswer (tested with Vivaldi)
3. Breaks in Firefox 66, too

## Expected result
![image](https://user-images.githubusercontent.com/996070/57701157-a7c02600-765b-11e9-9a29-bc4df3a9b033.png)
This is what I got with SemanticUI 2.4.x

## Actual result
![image](https://user-images.githubusercontent.com/996070/57701208-cd4d2f80-765b-11e9-9a1b-8effe50cfc57.png)

## Testcase

Fixed menu code. There's nothing more on the page. I made a test, and seems very specific of this icon: changing the icon solves the issue, and removing everything else in the menu does not.

```html
<div id="mainMenu" class="ui menu fixed purple inverted">
     <div id="item-mainMenu-0" class="no-active item">
          <img id="logo" src="/assets/images/logo-white.png" alt="LOGO">
     </div>
     <div id="item-mainMenu-3" class="no-active item left">
          <div id="dropdown-mainMenu-3" class="ui dropdown" value="Utilisateurs" tabindex="0">
               <div id="text-dropdown-mainMenu-3" class="text">Utilisateurs</div>
               <i id="dropdown-mainMenu-3-icon" class="icon dropdown"></i>
               <div class="menu transition hidden" tabindex="-1">
                    <a id="dd-item-dropdown-mainMenu-3-5" class="item" data-value="5">
                         <i id="" class="icon remove user"></i>Supprimer</a>
               </div>
          </div>
     </div>
</div>
```

## Screenshot (when possible)
Shown above.

## Version
v 2.7.2

**Comment by hammy2899** [comment]
This will most likely be a conflict with the `remove` icon class

**Comment by mlopezgva** [comment]
That's what I'm looking at right now.
`.ui.dropdown .remove.icon` is applied, but there's no `.ui.dropdown .edit.icon' ...

**Comment by lubber-de** [comment]
Duplicate of #622 
Fixed in upcoming 2.7.5
![removeusericondropdown](https://user-images.githubusercontent.com/18379884/57718877-c1269980-767e-11e9-8e61-392b329684db.gif)
